backend/src/v5/models/users.js

```diff
@@ -227,14 +226,14 @@ User.addUser = async (newUserData) => {
 		},
 		permissions: [],
 		...deleteIfUndefined({ sso: newUserData.sso }),
```

Comment:
I don't think you need `deleteIfFalse` or `deleteIfUndefined`
```suggestion
		...(newUserData.sso ? { sso: newUserData.sso } : { inactive: true } ),
```